scripts/package.json

```diff
@@ -63,7 +63,6 @@
     "markdown-table": "^2.0.0",
     "node-plop": "^0.25.0",
     "node-polyfill-webpack-plugin": "^1.0.2",
-    "node-sass": "^6.0.1",
```

Comment:
this is now a `devDependency` in root package.json (without the caret).
@layershifter - is that ok?

Reply:
Not Shift :) but we've generally been trying to move dev deps to the root and remove carets, so it should be fine.